dot/core/service_test.go

```diff
@@ -229,7 +228,7 @@ func Test_Service_handleCodeSubstitution(t *testing.T) {
 		},
 		"instance creation error": {
 			serviceBuilder: func(ctrl *gomock.Controller) *Service {
-				storedRuntime := NewMockInstance(ctrl)
+				storedRuntime := NewMockRuntimeInstance(ctrl)
```

Comment:
I think your branch should be rebased on development so these don't show as diffs right?

Reply:
Um I think it is still a diff? Renamed it to RuntimeInstance for clarity

Reply:
Oh yeah sorry I mixed up with `RuntimeInstance` from `dot/core` my bad